GVFS/GVFS/CommandLine/GVFSVerb.cs

```diff
@@ -884,31 +884,23 @@ private void CheckGitVersion(ITracer tracer, GVFSEnlistment enlistment, out stri
                 this.ReportErrorAndExit(tracer, "Error: Invalid version of git {0}.  Must use gvfs version.", version);
             }
 
-            if (ProcessHelper.IsDevelopmentVersion())
+            if (gitVersion.IsLessThan(GVFSConstants.SupportedGitVersion))
             {
-                if (gitVersion.IsLessThan(GVFSConstants.SupportedGitVersion))
-                {
-                    this.ReportErrorAndExit(
-                        tracer,
-                        "Error: Installed git version {0} is less than the supported version of {1}.",
-                        gitVersion,
-                        GVFSConstants.SupportedGitVersion);
-                }
-                else if (!gitVersion.IsEqualTo(GVFSConstants.SupportedGitVersion))
-                {
-                    this.Output.WriteLine($"Warning: Installed git version {gitVersion} does not match supported version of {GVFSConstants.SupportedGitVersion}.");
-                }
+                this.ReportErrorAndExit(
+                    tracer,
+                    "Error: Installed git version {0} is less than the supported version of {1}.",
+                    gitVersion,
+                    GVFSConstants.SupportedGitVersion);
             }
-            else
+            else if (gitVersion.Revision != GVFSConstants.SupportedGitVersion.Revision)

```

Comment:
I don't understand why we're having this `else if ... { ... }`.

I thought we were only enforcing a minimum installed Git version.  This reads like it is forcing an exact match.

Reply:
This is our escape hatch if we need to introduce a breaking change to `microsoft/git`: the version string `v2.XX.Y.vfs.Z.W` must match exactly only on the `.Z` part, but the rest is a minimum match.

Reply:
oh ok.  thanks